petric.py

```diff
@@ -134,7 +136,11 @@ def construct_RDP(penalty_strength, initial_image, kappa, max_scaling=1e-3):
     initial_image: used to determine a smoothing factor (epsilon).
     kappa: used to pass voxel-dependent weights.
     """
-    prior = STIR.RelativeDifferencePrior()
+    try:
+        prior = STIR.CudaRelativeDifferencePrior()
+    except:
```

Comment:
Do you know the exception type?


```suggestion
    except NameError:
```

Or something else?

Reply:
what does Python throw if it doesn't exist?